virtual_rainforest/main.py

```diff
@@ -232,13 +229,13 @@ def vr_run(
     # Identify models with shorter time steps than main loop and warn user about them
     check_for_fast_models(models_cfd, update_interval)
 
-    # TODO - Extract input data required to initialise the models
-
-    # TODO - Initialise the set of configured models
+    # TODO - A model spin up might be needed here in future
 
-    # TODO - Spin up the models
+    # Find parent folder for output for merge_file_path
+    parent_folder = merge_file_path.parent
 
-    # TODO - Save model state
+    # Save the initial state of the model
+    data.save_to_netcdf(Path(f"{parent_folder}/initial.nc"))
```

Comment:
This seems overkill - but it will be useful during testing. I think these names should be user configurable though and should be optional, so maybe look for`save_initial_data_path` and `save_final_data_path` or something similar in the configuration.

Ultimately, we may want to be able to specify which variables to save, but that can come later 😄

Reply:
Adding these as configuration options seems like a good idea!